livekit-agents/livekit/agents/voice/background_audio.py

```diff
@@ -0,0 +1,88 @@
+import asyncio
+from collections.abc import AsyncIterable
+from livekit import rtc
+from .agent_session import AgentSession
+from ..types import NotGiven, NOT_GIVEN, NotGivenOr
+from ..utils.audio import audio_frames_from_file
+from ..utils.aio import cancel_and_wait
+from ..utils import log_exceptions
+from ..log import logger
+
+from .events import AgentStateChangedEvent
+
+
+class BackgroundAudio:
```

Comment:
we should also offer a `play(path_to_file | audio_frames)` API.. that'll be more flexible because users might want to only play a specific audio triggered by certain function calls

Reply:
yeah 100%